From: =?utf-8?q?Rog=C3=A9rio_Brito?= <rbrito@gmail.com>
Date: Sun, 21 Feb 2021 22:29:12 -0300
Subject: Makefile: Install emacs.rc as a non-dotfile under /usr/share.

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 4ab445d..aa7df33 100644
--- a/Makefile
+++ b/Makefile
@@ -74,9 +74,9 @@ clean:
 install: $(PROGRAM)
 	cp em ${BINDIR}
 	cp emacs.hlp ${LIBDIR}
-	cp emacs.rc ${LIBDIR}/.emacsrc
+	cp emacs.rc ${LIBDIR}
 	chmod 755 ${BINDIR}/em
-	chmod 644 ${LIBDIR}/emacs.hlp ${LIBDIR}/.emacsrc
+	chmod 644 ${LIBDIR}/emacs.hlp ${LIBDIR}/emacs.rc
 
 lint:	${SRC}
 	@rm -f lintout
