From: =?utf-8?q?Rog=C3=A9rio_Brito?= <rbrito@ime.usp.br>
Date: Mon, 22 Feb 2021 00:03:14 -0300
Subject: debian/rules: Try to enable hardening to work.

---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 9297625..d7ff172 100644
--- a/Makefile
+++ b/Makefile
@@ -33,8 +33,8 @@ HDR=ebind.h edef.h efunc.h epath.h estruct.h evar.h util.h version.h
 # DO NOT ADD OR MODIFY ANY LINES ABOVE THIS -- make source creates them
 
 CC=gcc
-WARNINGS=-Wall -Wstrict-prototypes
-CFLAGS=-O2 $(WARNINGS) -g
+WARNINGS+=-Wall -Wstrict-prototypes
+CFLAGS+=-O2 $(WARNINGS) -g
 #CC=c89 +O3			# HP
 #CFLAGS= -D_HPUX_SOURCE -DSYSV
 #CFLAGS=-O4 -DSVR4		# Sun
@@ -50,10 +50,10 @@ ifeq ($(uname_S),Darwin)
 endif
 #DEFINES=-DAUTOCONF
 #LIBS=-ltermcap			# BSD
-LIBS=-lcurses			# SYSV
+LIBS+=-lcurses			# SYSV
 #LIBS=-ltermlib
 #LIBS=-L/usr/lib/termcap -ltermcap
-LFLAGS=-hbx
+LFLAGS+=-hbx
 BINDIR=$(DESTDIR)/usr/bin
 LIBDIR=$(DESTDIR)/usr/share/uemacs
 
